bundles/org.openhab.binding.enocean/README.md

```diff
@@ -255,11 +258,12 @@ The channels of a thing are determined automatically based on the chosen EEP.
 | repeaterMode        | String             | Set repeater level to 1, 2 or disable |
 | setBaseId           | String             | Changes the BaseId of your gateway. This can only be done 10 times! So use it with care. |
 | pushButton          | Trigger            | Channel type system:rawbutton, emits PRESSED and RELEASED events |
+| pushButton2         | Trigger            | Channel type system:rawbutton, emits PRESSED and RELEASED events |
 | doublePress         | Trigger            | Channel type system:rawbutton, emits PRESSED |
 | longPress           | Trigger            | Channel type system:rawbutton, emits PRESSED and RELEASED events |
 | rockerswitchA/B     | Trigger            | Channel type system:rawrocker, emits DIR1_PRESSED, DIR1_RELEASED, DIR2_PRESSED, DIR2_RELEASED events |
-| windowHandleState   | String             | Textual representation of handle position (OPEN, CLOSED, TILTED) |
-| windowSashState     | String             | Textual representation of sash position (OPEN, CLOSED, TILTED) |
+| windowHandleState   | String             | Textual representation of handle position (OPEN, CLOSED, TILTED or UP, DOWN, LEFT, RIGHT for the D2_06_01 EEP) |
+| windowSashState     | String             | Textual representation of sash position (OPEN, CLOSED, TILTED or TILTED, NOT TILTED for the D2_06_01 EEP) |
```

Comment:
In the description, maybe reorder it with something like: ths, and this for the D2_06_01 EEP and this, this this for all others